drivers/filesystems/CMakeLists.txt

```diff
@@ -2,12 +2,14 @@
 add_subdirectory(btrfs)
 add_subdirectory(cdfs)
 add_subdirectory(ext2)
-add_subdirectory(fastfat_new)
+add_subdirectory(fastfat)
 add_subdirectory(fs_rec)
 add_subdirectory(msfs)
 add_subdirectory(mup)
 add_subdirectory(nfs)
 add_subdirectory(npfs)
 add_subdirectory(ntfs)
 add_subdirectory(udfs)
-add_subdirectory(vfatfs)
+if(SARCH STREQUAL "xbox")
```

Comment:
This was made to compile for everything also for a reason....

Reply:
Double that.

Reply:
> This was made to compile for everything also for a reason....

What is that reason?
It seems to me to be wasting resources/time on usual builds.

Reply:
Judging from that logic we shouldn't add new code and new modules to avoid getting build times bigger. As for the reason, look at discussion/review of the FATX PR.

Reply:
> Judging from that logic we shouldn't add new code and new modules to avoid getting build times bigger.

I never wrote such a thing.
My point is: why compile something which is not going to be packaged?

> As for the reason, look at discussion/review of the FATX PR.

That PR seems to be  #4694.
Always-compile was explicitly intentional.
Nonetheless, the reason is still not obvious to me.

Anyway...

Reply:
I do concur that it would make sense to exclude that from the build when not needed.

Reply:
I don't want to fight neither. I just expressed my vote here, concuring Serges. I did neither reopen anything nor created a PR myself. It is jsut expressing my opinion. Which is as good or as bad as anybody elses opinion. In FOSS projects I do perceive that as important, to let democracy / swarm-intelligence find the best solutions/answers.

Reply:
As Xbox port maintainer I'd say... ideally ReactOS should be able to mount, read, and write FATX partitions regardless of being booted from Xbox or not.